stdlib/websub/src/main/ballerina/src/websub/hub_service.bal

```diff
@@ -26,6 +27,9 @@ import ballerina/time;
 
 map<PendingSubscriptionChangeRequest> pendingRequests = {};
 
+# This cache is used for caching HTTP clients against the subscriber callbacks.
+cache:Cache subscriberCallbackClientCache = new;
```

Comment:
The default expiry time seems to be 900000 milliseconds. 

Should we may be increase this?

Reply:
Increased for two days